static/app/components/links/link.tsx

```diff
@@ -2,14 +2,34 @@ import {
   Link as RouterLink,
   type LinkProps as ReactRouterLinkProps,
 } from 'react-router-dom';
+import isPropValid from '@emotion/is-prop-valid';
+import {css, type Theme} from '@emotion/react';
 import styled from '@emotion/styled';
 import type {LocationDescriptor} from 'history';
 
 import {locationDescriptorToTo} from 'sentry/utils/reactRouter6Compat/location';
 import normalizeUrl from 'sentry/utils/url/normalizeUrl';
 import {useLocation} from 'sentry/utils/useLocation';
 
-import {linkStyles} from './styles';
+export const linkStyles = ({disabled, theme}: {theme: Theme; disabled?: boolean}) => css`
+  /* @TODO(jonasbadalic) This was defined on theme and only used here */
+  border-radius: 2px;
+
+  &:focus-visible {
+    box-shadow: ${theme.linkFocus} 0 0 0 2px;
+    text-decoration: none;
+    outline: none;
+  }
+
+  ${disabled &&
+  css`
+    color: ${theme.disabled};
+    pointer-events: none;
+    :hover {
+      color: ${theme.disabled};
+    }
+  `}
```

Comment:
should we render a disabled link as a `span` ?

Reply:
I think so. I want to actually just try and remove disabled to see how usages we have and see if we can remove some